### PR TITLE
add ingest policy info to data collection docs

### PR DESCRIPTION
**Pull request by susanev**
### :nut_and_bolt: Description: What code changed, and why?
recently, in a [community slack convo](https://chefcommunity.slack.com/archives/CAZSL536K/p1592595733202400) there were some questions around what policy to add the api token to in the data collection docs, this specifies that folks should use the ingest policy.

### :chains: Related Resources
zero.

### :+1: Definition of Done
docs still work but are more clear

### :athletic_shoe: How to Build and Test the Change
`make serve` from `components/automate-chef-io`

### :white_check_mark: Checklist

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] Tests added/updated?
- [x] Docs added/updated?
- [x] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### :camera: Screenshots, if applicable
zero.

### CODE REVIEW

File: components/automate-chef-io/content/docs/data-collection.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ There are two steps to getting data collection running in Chef Automate:
 
 1. You must first have an API token. You have two options:
 
-  * [Create a new API Token]({{< relref "api-tokens.md#creating-api-tokens" >}}).
+  * [Create a new API Token]({{< relref "api-tokens.md#creating-api-tokens" >}}) and add it to the Ingest policy.
   * Or you can [use your existing data collector token]({{< relref "#existing-data-collector-token-setup" >}}) if you are migrating from Chef Automate 1.
 
 1. Once you have an API token, you can either:
```
Comment (mjingle) [inline components/automate-chef-io/content/docs/data-collection.md:27]: NTS: This numbering format needs to be fixed. Shows two `1.` in a row.

File: components/automate-chef-io/content/docs/data-collection.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ There are two steps to getting data collection running in Chef Automate:
 
 1. You must first have an API token. You have two options:
 
-  * [Create a new API Token]({{< relref "api-tokens.md#creating-api-tokens" >}}).
+  * [Create a new API Token]({{< relref "api-tokens.md#creating-api-tokens" >}}) and add it to the Ingest policy.
```
Comment (mjingle) [inline components/automate-chef-io/content/docs/data-collection.md:24]: @susanev How does this addition of "add it to the Ingest policy" play into the second step below of "Once you have an API token, you can either:"? Also, is there documentation for the Ingest policy that can be linked here?

File: components/automate-chef-io/content/docs/data-collection.md (hunk 1 above)
Comment (mjingle) [reply components/automate-chef-io/content/docs/data-collection.md:27]: @susanev I'm happy to fix this, but I need to leave myself a note so I don't forget! :)

File: components/automate-chef-io/content/docs/data-collection.md (hunk 2 above)
Comment (susanev) [reply components/automate-chef-io/content/docs/data-collection.md:24]: @mjingle the 2nd bullet point will automatically add it to the ingest policy for you, if you create an api token you'll need to choose the ingest policy in the dropdown in the create modal.

there's not really separate documentation on each chef-managed policy, other than the table of roles that are associated with the policy. we could link to that i suppose... https://automate.chef.io/docs/roles/#chef-managed-roles

Comment (mjingle) [reply components/automate-chef-io/content/docs/data-collection.md:24]: @susanev So then, this addition is clarifying a requirement to apply during the API Token creation?

File: components/automate-chef-io/content/docs/data-collection.md (hunk 1 above)
Comment (mjingle) [reply components/automate-chef-io/content/docs/data-collection.md:27]: I'll fix this in a separate PR!

File: components/automate-chef-io/content/docs/data-collection.md (hunk 2 above)
Comment (mjingle) [reply components/automate-chef-io/content/docs/data-collection.md:24]: If so, then I would recommend this rewording:
```suggestion
  * [Create a new API token]({{< relref "api-tokens.md#creating-api-tokens" >}}) and assign the API token to the Ingest policy during creation.
```

Comment (susanev) [reply components/automate-chef-io/content/docs/data-collection.md:24]: @mjingle I think _add_ is the right work here not _assign_ (at least thats the word used in the UI), and they dont necessarily have to do it during creation, but they can do it then.

Comment (mjingle) [reply components/automate-chef-io/content/docs/data-collection.md:24]: @susanev How about now?
```suggestion
  * [Create a new API token]({{< relref "api-tokens.md#creating-api-tokens" >}}) and add the API token to the Ingest policy, preferably at time of creation.
```

**Pull request merged**